python/tests/test_sklearn_vw.py

```diff
@@ -184,6 +186,20 @@ def test_decision_function(self, data):
 
         assert np.allclose(expected, actual)
 
```

Comment:
would be good to see a test of normal list input, something like this
```
    def test_shuffle_list(self):
        # dummy data in vw format
        X = ['1 |Pet cat', '-1 |Pet dog', '1 |Pet cat', '1 |Pet cat']

        # Classifier with multiple passes over the data
        clf = VWClassifier(passes=3, convert_to_vw=False)
        clf.fit(X)

        # assert that the dummy data was not perturbed
        assert X == ['1 |Pet cat', '-1 |Pet dog', '1 |Pet cat', '1 |Pet cat']
```

Reply:
Makes sense. Copied over the test suggestion.